### PR TITLE
NMS-7472: Fix various problems in ResponseTimeSummary_Availability_Offenders_subreport

### PR DESCRIPTION
**Pull request by soleger**
Check out the JIRA issue for the problems that are addressed.

* JIRA: http://issues.opennms.org/browse/NMS-7472
* Bamboo: http://bamboo.internal.opennms.com:8085/browse/OPENNMS-ONMS796

### CODE REVIEW

File: opennms-base-assembly/src/main/filtered/etc/report-templates/subreports/ResponseTimeSummary_Availability_Offenders_subreport.jrxml [hunk 1]
```diff
@@ -13,13 +13,28 @@
...
-		<defaultValueExpression><![CDATA["7 Days"]]></defaultValueExpression>
+	<parameter name="DS_SURVEILLANCE_CATEGORY" class="java.lang.String">
+		<parameterDescription><![CDATA[Surveillance category to include in this report]]></parameterDescription>
+	</parameter>
+	<parameter name="TIME_RANGE_NUMBER" class="java.lang.Integer">
+		<parameterDescription><![CDATA[Report Time Range in Days]]></parameterDescription>
+		<defaultValueExpression><![CDATA[7]]></defaultValueExpression>
+	</parameter>
+	<parameter name="START_TIME" class="java.util.Date">
+		<defaultValueExpression><![CDATA[new java.util.Date(new java.util.Date().getTime() - (86400000L * $P{TIME_RANGE_NUMBER}))]]></defaultValueExpression>
```
Comment (mvrueden) [inline opennms-base-assembly/src/main/filtered/etc/report-templates/subreports/ResponseTimeSummary_Availability_Offenders_subreport.jrxml]: This parameter is going to show up in the database reports web ui when selected. 
We should probably add a description.

Comment (soleger) [inline opennms-base-assembly/src/main/filtered/etc/report-templates/subreports/ResponseTimeSummary_Availability_Offenders_subreport.jrxml]: Do you mean the START_TIME parameter? I can add a short descr for it.

Comment (mvrueden) [inline opennms-base-assembly/src/main/filtered/etc/report-templates/subreports/ResponseTimeSummary_Availability_Offenders_subreport.jrxml]: Yes. 
Each paramater not marked as `isForPrompting=false` is showing up. 
If a description is defined, a tooltip is shown to the user in the web ui.

**Comment by soleger** [comment]
These changes have been merged to foundation-2016. Closing PR.

**Pull request closed without merging**